dashboard/serializers_test.py

```diff
@@ -28,13 +45,23 @@ class UserProgramSearchSerializerTests(TestCase):
     """
     Test cases for the UserProgramSearchSerializer
     """
+
     @staticmethod
     def _generate_cached_enrollments(user, program, num_course_runs=1, data=None):
         """
         Helper method to generate CachedEnrollments for test cases
         """
+        fake = faker.Factory.create()
         course = CourseFactory.create(program=program)
-        course_runs = [CourseRunFactory.create(course=course) for _ in range(num_course_runs)]
+        course_runs = [
+            CourseRunFactory.create(
+                course=course,
+                enrollment_start=fake.date_time_this_month(before_now=True, after_now=False, tzinfo=pytz.utc),
+                start_date=fake.date_time_this_month(before_now=True, after_now=False, tzinfo=pytz.utc),
+                enrollment_end=fake.date_time_this_month(before_now=True, after_now=False, tzinfo=pytz.utc),
+                end_date=fake.date_time_this_year(before_now=True, after_now=False, tzinfo=pytz.utc),
```

Comment:
can you set these params (`before_now=True, after_now=False, tzinfo=pytz.utc`) to a dict, and invoke `fake.date_time_this_year` by passing in that dict? something like `fake.date_time_this_month(**current_date_params)`